plugins/rapid7_insightidr/komand_rapid7_insightidr/actions/get_alert_actors/action.py

```diff
@@ -0,0 +1,24 @@
+import insightconnect_plugin_runtime
+from .schema import GetAlertActorsInput, GetAlertActorsOutput, Input, Output, Component
+
+# Custom imports below
+from komand_rapid7_insightidr.util.endpoints import Alerts
+from komand_rapid7_insightidr.util.resource_helper import ResourceHelper
+
+
+class GetAlertActors(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="get_alert_actors",
+            description=Component.DESCRIPTION,
+            input=GetAlertActorsInput(),
+            output=GetAlertActorsOutput(),
+        )
+
+    def run(self, params={}):
```

Comment:
If the other two inputs aren't needed for this action then would it be better to remove them?
From schema:
```
class Input:
    ALERT_RRN = "alert_rrn"
    INDEX = "index"
    SIZE = "size"
```